src/components/ReviewPage.jsx

```diff
@@ -24,11 +28,40 @@ export default function ReviewPage() {
 		});
 	}, []);
 
-	const handleReviewUpVoteClick = () => {};
-	const handleReviewDownVoteClick = () => {};
-	const handleCommentUpVoteClick = () => {};
-	const handleCommentDownVoteClick = () => {};
-	const handleCommentDelete = () => {};
+	const handleReviewUpVoteClick = () => {
+		if(review.owner !== user.username){
```

Comment:
although I like this idea maybe give the user some feedback as to whats happening when they try to upvote their own post.